.github/scripts/release-against-charts.sh

```diff
@@ -19,44 +19,52 @@ if [ ! -e ~/.gitconfig ]; then
     git config --global user.email fleet@suse.de
 fi
 
-if [ ! -f bin/charts-build-scripts ]; then
-    make pull-scripts
-fi
-
-if grep -q "version: ${PREV_CHART_VERSION}" ./packages/fleet/fleet/package.yaml && grep -q "${PREV_FLEET_VERSION}" ./packages/fleet/fleet/package.yaml; then
-    find ./packages/fleet/ -type f -exec sed -i -e "s/${PREV_FLEET_VERSION}/${NEW_FLEET_VERSION}/g" {} \;
-    find ./packages/fleet/ -type f -exec sed -i -e "s/version: ${PREV_CHART_VERSION}/version: ${NEW_CHART_VERSION}/g" {} \;
+if [[ "$(git rev-parse --abbrev-ref HEAD)" =~ dev-v2\.1[0-9]+ ]]; then
```

Comment:
Could these changes also be applied to `.github/scripts/release-against-test-charts.sh`?